environments.json

```diff
@@ -1284,14 +1284,14 @@
     "family": "V8",
     "short": "CH&nbsp;59,<br>OP&nbsp;46",
     "note_id": "experimental-flag",
-    "obsolete": false
+    "obsolete": true
```

Comment:
Is Chrome 59 really obsolete, just because 60 is available? My Chrome is still on 59, and it took me explicitly checking the version to initiate an update.